compose/ui/ui/src/skikoTest/kotlin/androidx/compose/ui/window/PopupTest.kt

```diff
@@ -453,4 +459,48 @@ class PopupTest {
         scene.sendPointerEvent(PointerEventType.Release, Offset(11f, 11f), button = PointerButton.Primary)
         background.events.assertReceivedNoEvents()
     }
+
+    @Test
+    fun secondClickDoesNotDismissPopup() = runSkikoComposeUiTest(
```

Comment:
I don't think the best place to test the low-level touch dispatching is by examining popup behavior. There should be a test for the low-level dispatching by itself.

Reply:
It just follows the existing structure. It was like that because Popup was the single usage of it